lib/function_loader.js

```diff
@@ -32,6 +32,7 @@ function checkConflicts(obj1, obj2) {
 module.exports = function(eyeglass, sass, options, existingFunctions) {
   var root = options.root;
   var functions = {};
+  hash.merge(functions, require("./functions")(eyeglass, sass));
```

Comment:
We probably want to merge our "protected" functions at the end. It would also allow us to offer better errors/warnings if it's a collision vs overriding one of eyeglass' native methods